packages/framework/src/Resources/config/admin_menu.yml

```diff
@@ -180,98 +180,88 @@
           route: admin_payment_edit
           visible: false
 
-    - label: 'Communication with customer'
+    - label: 'E-mail settings'
       route: admin_mail_setting
-      items:
-        - label: 'E-mail settings'
-          route: admin_mail_setting
 
-        - label: 'E-mail templates'
-          route: admin_mail_template
+    - label: 'E-mail templates'
+      route: admin_mail_template
 
-        - label: 'Order confirmation page'
-          route: admin_customercommunication_ordersubmitted
+    - label: 'Order confirmation page'
+      route: admin_customercommunication_ordersubmitted
 
-    - label: 'Lists and nomenclatures'
+    - label: 'Availability'
       route: admin_availability_list
+
+    - label: 'Flags'
+      route: admin_flag_list
+
+    - label: 'Parameters'
+      route: admin_parameter_list
+
+    - label: 'Status of orders'
+      route: admin_orderstatus_list
+      items:
+        - label: 'Editing order status'
+          route: admin_orderstatus_edit
+          visible: false
+        - label: 'New orders status'
+          route: admin_orderstatus_new
+          visible: false
+
+    - label: 'Brands'
+      route: admin_brand_list
       items:
-        - label: 'Availability'
-          route: admin_availability_list
-
-        - label: 'Flags'
-          route: admin_flag_list
-
-        - label: 'Parameters'
-          route: admin_parameter_list
-
-        - label: 'Status of orders'
-          route: admin_orderstatus_list
-          items:
-            - label: 'Editing order status'
-              route: admin_orderstatus_edit
-              visible: false
-            - label: 'New orders status'
-              route: admin_orderstatus_new
-              visible: false
-
-        - label: 'Brands'
-          route: admin_brand_list
-          items:
-            - label: 'Editing brand'
-              route: admin_brand_edit
-              visible: false
-            - label: 'New brand'
-              route: admin_brand_new
-              visible: false
-
-        - label: 'Units'
-          route: admin_unit_list
-
-        - label: 'Countries'
-          route: admin_country_list
+        - label: 'Editing brand'
+          route: admin_brand_edit
+          visible: false
```

Comment:
The items are there because of Breadcrumb. There was always some hard-coded maximum level. Previously only second level was rendered in the left menu, now there have been two levels. Changed to three levels as I've dropped the commit based on your and @LukasHeinz's feedback.

This is something we will probably address in the future.